src/ProjectionMapping/ProjectionDeckMap/layers.js

```diff
@@ -38,8 +38,20 @@ export const createGeoJsonLayer = (i, layer, GEOGRID) =>
     extruded: true,
     lineWidthScale: 20,
     lineWidthMinPixels: 2,
-    getLineColor: [255, 255, 255],
-    getFillColor: [200, 200, 200],
+    getLineColor: f => {
+      const hex = f.properties.color;
+      if (!hex) return [0, 0, 0];
+    
+      const rgba = hex.match(/[0-9a-f]{2}/gi).map(x => parseInt(x, 16));
+      return rgba.length === 4 ? rgba : rgba.slice(0, 3);
+    },
+    getFillColor: f => {
+      const hex = f.properties.color;
```

Comment:
It’d be nice if we could have two separate color definitions: f.properties.fillColor and f.properties.lineColor, in case we need to use different colors